src/main/java/ru/hse/sd/roguegue/UI/MapUI.java

```diff
@@ -0,0 +1,57 @@
+package ru.hse.sd.roguegue.UI;
+
+import ru.hse.sd.roguegue.status.Constants;
+import ru.hse.sd.roguegue.map.CellType;
+import ru.hse.sd.roguegue.state.Position;
+import ru.hse.sd.roguegue.status.Status;
+
+import java.util.EnumMap;
+
+/**
+ * Class for displaying the map. It should have graphic representations for all cell types.
+ * Can display a full map or a one cell.
+ */
+public class MapUI {
+    private final EnumMap<CellType, Character> graphicRepresentation;
+    private final int dx = Constants.MAP_START_X;
+    private final int dy = Constants.MAP_START_Y;
+
+
+    /**
```

Comment:
```suggestion
    private final int dy = Constants.MAP_START_Y;

    /**
```
Придирка, конечно, но две пустые строки подряд обычно не делают